specification/ai/DocumentIntelligence/tspconfig.yaml

```diff
@@ -18,11 +18,11 @@ options:
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/DocumentIntelligence.json"
   # Uncomment this line and add "@azure-tools/typespec-python" to your package.json to generate Python code
   "@azure-tools/typespec-python":
-    "basic-setup-py": true
-    "package-version": 1.0.0b1
-    "package-dir": "azure-ai-documentintelligence"
-    "package-name": "azure-ai-documentintelligence"
-    # "output-path":
+    package-pprint-name: "\"Azure AI Document Intelligence\""
+    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
```

Comment:
Also, just checking do you need this property?

Reply:
Let me try to generate without it.

Reply:
No difference, I'll remove this line.